src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -360,21 +361,44 @@ public void datasetsHappyPath() throws Exception {
     assertThrows(AssertionError.class, () -> dataRepoFixtures.deleteProfile(steward, profileId));
   }
 
-  private void ingestTable(String tableName, String ingestFile, long expectedRowCount)
-      throws Exception {
-    List<Map<String, Object>> data;
-    try {
-      data = jsonLoader.loadObjectAsStream(ingestFile, new TypeReference<>() {});
-    } catch (Exception e) {
-      throw new RuntimeException("Error building ingest request", e);
+  record IngestSource(String tableName, String ingestFile, int expectedRowCount) {}
```

Comment:
This code duplicates that in https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java#L298

The main difference is that this uses `DataRepoFixtures` and the GCP test uses `ConnectedOperations`. The tests are otherwise unrelated but I could refactor the common code if we think it makes sense to do so.